frb_codegen/src/library/codegen/config/internal_config_parser/mod.rs

```diff
@@ -128,6 +128,7 @@ impl InternalConfig {
                 add_mod_to_lib: config.add_mod_to_lib.unwrap_or(true),
                 build_runner: config.build_runner.unwrap_or(true),
                 web_enabled,
+                dart_output: dart_output_dir,
```

Comment:
nit: shall we name it `dart_output_dir` to keep it consistent

Reply:
`dart_output` has been declared before `dart_output_dir`. We could leave it as-is.

Reply:
https://github.com/fzyzcjy/flutter_rust_bridge/pull/2340/files#diff-e15babd1f228e841d85dd0a41104cb9dc93561d502b0854a82a9edcbdf6af5daR37

Reply:
Looks good!